docs/book/src/tasks/certs/experimental-ca-rotation.md

```diff
@@ -0,0 +1,112 @@
+- [ ] edit managed cluster's cluster-info ConfigMap to match pubCA 1+2
+- [ ] rollout restart all Workers pools
+- [ ] rollout restart all control-plane nodes
+- [ ] rollout restart all pods running in managed cluster
+
+**BIG-BANG LIVE OPERATIONS :**
+
+<aside class="note warning">
+
+<h1>Machine remote access is required</h1>
+
+In order to manually replace CA and renew all control-plane certificates at once, this procedure requires to log into systems using SSH. So it breaks CAPI *"no direct access to the machines"* concept and (by nature) will never be generalized nor automated "as it is" by any CAPI controller.
+
+This is only a manual / workaround procedure and should be taken as it is.
+
+</aside>
+
+- [ ] **pause CAPI cluster reconciliation**
+- [ ] replace key & crt files on all control planes & renew cert **on all control-plane nodes at the same time**:
+
+```bash
+# as root
+# kube CA
+echo -n "<new CA private key content>" > /etc/kubernetes/pki/ca.key
+chmod 0600 /etc/kubernetes/pki/ca.key
+
+echo -n "<new CA pubcert + old CA pubcert content>" > /etc/kubernetes/pki/ca.crt
+chmod 0644 /etc/kubernetes/pki/ca.crt
+
+echo -n "<new CA public key content>" > /etc/kubernetes/pki/cluster-signing-ca.crt
+chmod 0644 /etc/kubernetes/pki/cluster-signing-ca.crt
+
+# kube PROXY
+echo -n "<new PROXY private key content>" > /etc/kubernetes/pki/front-proxy-ca.key
+chmod 0600 /etc/kubernetes/pki/front-proxy-ca.key
+
+echo -n "<new PROXY pubcert + old PROXY pubcert content>" > /etc/kubernetes/pki/front-proxy-ca.crt
+chmod 0644 /etc/kubernetes/pki/front-proxy-ca.crt
+
+# kube ETCD
+echo -n "<new ETCD private key content>" > /etc/kubernetes/pki/etcd/ca.key
+chmod 0600 /etc/kubernetes/pki/etcd/ca.key
+
+echo -n "<new ETCD pubcert + old ETCD pubcert content>" > /etc/kubernetes/pki/etcd/ca.crt
+chmod 0644 /etc/kubernetes/pki/etcd/ca.crt
+
+# regen certs on all VM at the same time !
+kubeadm certs renew all
+
+crictl stop $(crictl ps |grep kube-apiserver |cut -d' ' -f1)
+crictl stop $(crictl ps |grep kube-controller-manager |cut -d' ' -f1)
+crictl stop $(crictl ps |grep kube-scheduler |cut -d' ' -f1)
+
+# fetch admin token to fill it back in CAPI cluster...
+cat /etc/kubernetes/admin.conf
+```
+
+- [ ] deploy CAPI cluster CA Secret pub 2 / key 2
+- [ ] deploy CAPI cluster PROXY Secret pub 2 / key 2
+- [ ] update CAPI kubeconfig admin Secret with new admin.conf kubeconfig content using admin.conf
+- [ ] edit managed cluster's cluster-info ConfigMap to only expose pubCA 2
+- [ ] **resume CAPI cluster reconciliation**
+- [ ] rollout restart all Worker machines
+- [ ] rollout restart all CP machines
+- [ ] (optional) rollout restart all managed cluster's pods
+
+### Rotating `cluster-sa` keypair
+
+- *TODO*
```

Comment:
Are you planning to include this in this PR?

Reply:
It's not planned at the moment.
We can perfectly remove it to avoid confusion